src/main/java/org/spongepowered/api/entity/living/Humanoid.java

```diff
@@ -122,4 +127,44 @@ default FoodData getFoodData() {
         return getValue(Keys.SATURATION).get();
     }
 
+    /**
+     * Gets this {@link Humanoid}'s textures.
+     *
+     * @return The textures property
+     */
+    Optional<ProfileProperty> getTextures();
+
+    /**
+     * Gets this {@link Humanoid}'s textures, as seen by the viewer.
+     *
+     * @param viewer The viewer of the request
+     * @return The textures property, as seen by the viewer
+     */
+    Optional<ProfileProperty> getTextures(@Nullable ContextViewer viewer);
+
+    /**
+     * Sets the fake name and textures
+     *
+     * @param viewer The viewer of the request
+     * @param name The fake name
+     * @param property The textures property
+     * @return The transaction result
+     */
+    DataTransactionResult setFakeNameAndTextures(ContextViewer viewer, String name, ProfileProperty property);
```

Comment:
If this affects `getTextures(@Nullable ContextViewer viewer)`, it should probably be named `setTextures` for consistency.